include/envoy/compressor/compressor.h

```diff
@@ -13,11 +18,11 @@ class Compressor {
   virtual ~Compressor() {}
 
   /**
-   * Compresses data from one buffer into another buffer.
-   * @param input_buffer supplies the buffer with data to be compressed.
-   * @param output_buffer supplies the buffer to output compressed data.
+   * Compresses data buffer.
+   * @param buffer supplies the reference to data to be compressed.
```

Comment:
Can you specify that the buffer contents will be replaced inline with compressed data?